contracts/margin/external/exchangewrappers/ZeroExV2MultiOrderExchangeWrapper.sol

```diff
@@ -171,15 +176,56 @@ contract ZeroExV2MultiOrderExchangeWrapper is
         total.takerAmount = 0;
         total.makerAmount = desiredMakerToken;
 
+        // gets the exchange cost. modifies total
+        getExchangeCostInternal(
+            makerToken,
+            orders,
+            total
+        );
+
+        // require that all amount was bought
```

Comment:
wording is weird. Maybe "require that the entire amount was bought"

Reply:
done